index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Onomeh's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins:regular,bold,500,600&amp;display=swap"
+        />
+      <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+        <header class="tool-bar">
+          <div class="tools">
+            <a href="/" class="logo">Onomeh </a>
+            <nav class="navbar">
+              <button class="nav-button" type="button">
+                <img src="assets/Icons/hamburger-button.svg" alt="more" />
+              </button>
```

Comment:
- [ ] You did a very great job on the design but there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your hamburger icon should be on the right of the screen but currently, it is appearing in the center. Please place your hamburger icon right side of the screen and make sure it matches the design. 

Your design

![Screenshot (153)](https://user-images.githubusercontent.com/83803414/154527409-5b45fac5-fa21-4e2a-bbff-fa2cc425bcfd.png)

Figma design

![Screenshot (152)](https://user-images.githubusercontent.com/83803414/154527443-02fb194c-b0d0-4084-8a18-bbc564374f2e.png)